routes/accessControl.js

```diff
@@ -0,0 +1,25 @@
+var tokenHelper = require('../util/shared/tokenHelper.js');
+var util = require('../util/shared/util.js');
+
+
+module.exports = function(app){
+	app.use('/api/*', function(req, res, next) {
+		console.log('inside ac');
+
+		var tokenString = req.body.token;
```

Comment:
[b]The token should be part of header of the request, not the body here.
You should not ask every request to be post and have a body field "token".